### PR TITLE
fix(planner_manager): fix build error

### PR DESCRIPTION
**Pull request by satoshi-ota**
## Description

Fix build error on new manager.

- https://github.com/autowarefoundation/autoware.universe/pull/3605

```
--- stderr: behavior_path_planner                                                                                                                                                                                                                                                                                                                                                                                               
CMake Warning at CMakeLists.txt:78 (message):                                                                                                                                                                                                                                                                                                                                                                                   
  Build behavior_path_planner with NEW framework...                                                                                                                                                                                                                                                                                                                                                                             
                                                                                                                                                                                                                                                                                                                                                                                                                                
                                                                                                                                                                                                                                                                                                                                                                                                                                
/home/satoshi/pilot-auto/src/autoware/universe/planning/behavior_path_planner/src/planner_manager.cpp: In lambda function:                                                                                                                                                                                                                                                                                                      
/home/satoshi/pilot-auto/src/autoware/universe/planning/behavior_path_planner/src/planner_manager.cpp:51:7: error: ‘scene_modules_’ was not declared in this scope                                                                                                                                                                                                                                                              
   51 |       scene_modules_.begin(), scene_modules_.end(),                                                                                                                                                                                                                                                                                                                                                                     
      |       ^~~~~~~~~~~~~~                                                                                                                                                                                                                                                                                                                                                                                                    
/home/satoshi/pilot-auto/src/autoware/universe/planning/behavior_path_planner/src/planner_manager.cpp: In lambda function:                                                                                                                                                                                                                                                                                                      
/home/satoshi/pilot-auto/src/autoware/universe/planning/behavior_path_planner/src/planner_manager.cpp:52:70: error: ‘BT’ has not been declared                                                                                                                                                                                                                                                                                  
   52 |       [](const auto & module) { return module->getCurrentStatus() == BT::NodeStatus::RUNNING; });                                                                                                                                                                                                                                                                                                                       
      |                                                                      ^~                                                                                                                                                                                                                                                                                                                                                 
gmake[2]: *** [CMakeFiles/behavior_path_planner_node.dir/build.make:76: CMakeFiles/behavior_path_planner_node.dir/src/planner_manager.cpp.o] Error 1                                                                                                                                                                                                                                                                            
gmake[2]: *** Waiting for unfinished jobs....                                                                                                                                                                                                                                                                                                                                                                                   
gmake[1]: *** [CMakeFiles/Makefile2:186: CMakeFiles/behavior_path_planner_node.dir/all] Error 2                                                                                                                                                                                                                                                                                                                                 
gmake: *** [Makefile:146: all] Error 2                                                                                                                                                                                                                                                                                                                                                                                          
---                                                                                                                                                                                                                                                                                                                                                                                                                             
Failed   <<< behavior_path_planner [3min 16s, exited with code 2]
```

<!-- Write a brief description of this PR. -->

## Tests performed

<!-- Describe how you have tested this PR. -->
<!-- Although the default value is set to "Not Applicable.", please update this section if the type is either [feat, fix, perf], or if requested by the reviewers. -->

Build passed in my local pc.

## Effects on system behavior

<!-- Describe how this PR affects the system behavior. -->

Nothing.

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [x] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: planning/behavior_path_planner/src/planner_manager.cpp [hunk 1]
```diff
@@ -47,20 +47,22 @@ BehaviorModuleOutput PlannerManager::run(const std::shared_ptr<PlannerData> & da
...
 
   auto result_output = [&]() {
-    const bool is_any_module_running = std::any_of(
-      scene_modules_.begin(), scene_modules_.end(),
-      [](const auto & module) { return module->getCurrentStatus() == BT::NodeStatus::RUNNING; });
-    if (
-      !is_any_module_running &&
-      utils::isEgoOutOfRoute(
-        data->self_odometry->pose.pose, data->prev_modified_goal, data->route_handler)) {
+    const auto is_any_module_running = std::any_of(
```
Comment (shmpwk) [inline planning/behavior_path_planner/src/planner_manager.cpp:50]: minor comment, isn' t it bool type?

Comment (kosuke55) [reply planning/behavior_path_planner/src/planner_manager.cpp:50]: (and `const auto is_out_of_route` )

**Review by kosuke55 — APPROVED** [review]
LGTM:bow:

File: planning/behavior_path_planner/src/planner_manager.cpp [hunk 2]
```diff
@@ -47,20 +47,22 @@ BehaviorModuleOutput PlannerManager::run(const std::shared_ptr<PlannerData> & da
...
+      approved_module_ptrs_.begin(), approved_module_ptrs_.end(),
+      [](const auto & m) { return m->getCurrentStatus() == ModuleStatus::RUNNING; });
+
+    const auto is_out_of_route = utils::isEgoOutOfRoute(
```
Comment (satoshi-ota) [inline planning/behavior_path_planner/src/planner_manager.cpp:54]:
```suggestion
    const bool is_out_of_route = utils::isEgoOutOfRoute(
```

File: planning/behavior_path_planner/src/planner_manager.cpp (hunk 1 above)
Comment (satoshi-ota) [inline planning/behavior_path_planner/src/planner_manager.cpp:50]:
```suggestion
    const bool is_any_module_running = std::any_of(
```

**Review by shmpwk — APPROVED** [review]

**Pull request merged**